supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -8,6 +8,21 @@ Follow the _IBM Style_ guide advice of _day Month year_ for date formats, for ex
 
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
+[[man-pages]]
+== Man page references
+
+When referencing a man page in an additional resources section, use the following format: `_<name_of_utility>(<section_number>)_` _man page on your system_.
+
+Do not link to a website containing the man page information. Users run the `man` command on their system to view the information for the named command or utility.
```

Comment:
Should we explain that the man-page content can vary from one system to another, and the user must run the `man` command on the system where the named command will be run, to ensure that they see the relevant system-specific information?

Reply:
I meant that I think it's useful to explain _why_ we're including the text "on your system" in the examples.

Reply:
If it's about adding the why to SSG (not to the actual entries), I agree that it would be helpful.

Reply:
If we're adding the why to the SSG, it is important to note that the content of man files is determined not by OS version, but by kernel version. You can have multiple kernel versions across the life of an OS Version.

Reply:
@libander what do you mean with "the content of man files is determined not by OS version, but by kernel version"?

Man pages are shipped with the RPMs of tools, services, etc. they belong to. For example, `find(1)` is provided by the `findutils` RPM, and `rsync(1)` by the `rsync` package. If you update one of these packages, then the man pages are also updated (but the man page content only changes if the new RPM also contains a different man page). Anyway, man pages don't change if you boot a different kernel.

None of the kernel packages (`kernel`, `kernel-core`, `kernel-modules`, `kernel-modules-extra`, `kernel-modules-core`) contains a man page. I don't know if there are other packages than `kernel*` that can be installed at the same time in different versions, but if such packages contain man pages, then the old man pages are replaced by the new ones because the files use the same paths to the man page files. If you check, for example, `/usr/share/man/man1/` you see that there's only one man page for each tool.

In case that you meant something else, can you please explain it further?

Reply:
> In case that you meant something else, can you please explain it further?

You're right, many packages come with their own man-page included. The [`man-pages.rpm`](https://access.redhat.com/downloads/content/rhel---9/x86_64/11051/man-pages/6.04-1.el9/src/fd431d51/package) provides man-pages for [GNU/Linux](https://git.kernel.org/pub/scm/docs/man-pages/man-pages.git/about/) system and library calls, etc. 

> adding the why to SSG (not to the actual entries)

It was with this context in mind I was suggesting that we be slightly more specific than 'vary by system'  -- 'vary by package or kernel version' would be less mysterious. :)